stratum/hal/lib/phal/sfp_adapter.h

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2019 
```

Comment:
Missing company

Reply:
Hmm looks like a merge problem.  thanks.  Should we also have the "Copyright 2018-present Open Networking Foundation" header?

Reply:
Well, depends on who wrote this code. If it's yours then go with DELL EMC, else `Copyright 2019 Open Networking Foundation` should do it.